pom.xml

```diff
@@ -26,13 +26,13 @@
     <properties>
         <revision>1.45</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.121.1</jenkins.version>
+        <jenkins.version>2.164.3</jenkins.version>
```

Comment:
2.164.1 is the oldest LTS available in parent-pom 4.x.
And 2.164.1 caused following test failure:
```
[ERROR] org.jvnet.hudson.test.PluginAutomaticTestBuilder$OtherTests.testPluginActive  Time elapsed: 0.037 s  <<< ERROR!
java.lang.Error: Plugin junit failed to start
        at org.jvnet.hudson.test.PluginAutomaticTestBuilder$OtherTests.testPluginActive(PluginAutomaticTestBuilder.java:99)
...
Caused by: java.io.IOException: JUnit Plugin version 1.29 failed to load.
 - You must update Jenkins from version 2.164.1 to version 2.164.3 or later to run this plugin.
        at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:821)
...
```

junit-plugin 1.29 is come from maven-plugin 3.1.2, and upgrading maven-plugin can be an option.